R/radab.R

```diff
@@ -58,7 +58,7 @@ radab <- function(adsl,
                   cached = FALSE) {
   checkmate::assert_flag(cached)
   if (cached) {
-    return(get_cached_data("cadpc"))
+    return(get_cached_data("cadab"))
```

Comment:
does that mean we will need to recompile everything downstream?

Reply:
@shajoezhu since we don't use this argument and get our cached data directly from `cadab` and not `radab(cached = TRUE)` this shouldn't affect us or anything downstream.